docs.kosli.com/content/faq/_index.md

```diff
@@ -242,3 +242,17 @@ kosli attest generic Dockerfile true ...
 ```
 The parser then sees `Dockerfile` and `true` as the two
 arguments to `kosli attest generic`.
+
+## Fingerprint calculation
```

Comment:
Maybe use a "more direct" section title? 
This is not really about the calculation - it is about the Artifact name.

Maybe "Path/Artifact is a single space character!"